airflow/providers/apache/druid/hooks/druid.py

```diff
@@ -144,13 +144,15 @@ def submit_indexing_job(
 
         sec = 0
         while running:
-            req_status = requests.get(druid_task_status_url, auth=self.get_auth())
+            req_status = requests.get(druid_task_status_url, auth=self.get_auth(), verify=self.get_verify())
```

Comment:
Note that the default is True in both the [requests library](https://github.com/psf/requests/blob/main/src/requests/api.py#L39-L41) and [the init](https://github.com/apache/airflow/blob/main/airflow/providers/apache/druid/operators/druid.py#L56) of the hook, meaning it's backwards compatible